config.yaml

```diff
@@ -39,10 +39,10 @@
             # Make sure that the samples you merge have the same sample features (e.g paired, sense, dup_type)
             # Choose an informative experiment name
   experiment_name: {
-                    rbp: 'rbp name', #rna binding protein name, eg: PUM2,
-                    replicates: ["fg_sample1"], # sample(s) to be treated as foreground.
+                    rbp: 'PUM2', #rna binding protein name, eg: PUM2,
+                    replicates: ["ENCFF041KJT_ENCFF462SCV"], # sample(s) to be treated as foreground.
                                                 # If more than one they will be merged and treated as one sample
-                    smis: ["bg_sample1"], # sample(s) to be treated as background.
+                    smis: ["ENCFF616FCF_ENCFF495ZPY"] , # sample(s) to be treated as background.
                                           # If more than one they will be merged and treated as one sample
                     window_f: 300, # size to split windows. The window size should be at least double the fragment size
```

Comment:
Not clear where these windows matter. Maybe "Size of the window to slide over the reference sequences to pre-screen for read enrichment. Can be different for the foreground and background samples."